marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/map-multiple-entities/writeEntityInstances.sjs

```diff
@@ -0,0 +1,42 @@
+const DataHubSingleton = require("/data-hub/5/datahub-singleton.sjs");
```

Comment:
Since this test is running a flow, let's name this test "runFlowWithRelatedEntityMapping.sjs" - "writeEntityInstances" doesn't imply that a flow is being run.